packages/gafl-webapp-service/src/pages/licence-details/licence-start-time/__tests__/result-function.spec.js

```diff
@@ -0,0 +1,33 @@
+import resultFunction from '../result-function'
+import { CommonResults } from '../../../../constants.js'
+
+describe('contact > result-function', () => {
+  const mockStatusCacheGet = jest.fn(() => ({}))
+
+  const mockRequest = {
+    cache: () => ({
+      helpers: {
+        status: {
+          getCurrentPermission: mockStatusCacheGet
+        }
+      }
+    })
+  }
```

Comment:
Same here - the pattern we use is to have a function to generate a mock request, not a reused `mockRequest` object